src/Network/Network/ChangeLog.md

```diff
@@ -25,6 +25,13 @@
     - `Update-AzVpnConnection`
 * Added new cmdlet to fetch IKE Security Associations for VPN Site Link Connections.
     - `Get-VpnSiteLinkConnectionIkeSa`
+* Added new cmdlet to reset a Virtual Network Gateway Connection.
+    - `Reset-AzVirtualNetworkGatewayConnection`
+* Added new cmdlet to reset a Vpn Site Link Connection.
+    - `Reset-VpnSiteLinkConnection`
```

Comment:
These Reset changes were added in PR https://github.com/Azure/azure-powershell/pull/14225 but ChangeLog in that PR was incorrect so fixed it.